pymatgen/io/abinit/pseudos.py

```diff
@@ -1024,7 +1024,7 @@ class ppdesc(NamedTuple):
         10: ppdesc(10, "HGHK", "NC", None),
     }
 
-    del ppdesc
+    # del ppdesc
```

Comment:
Same to https://github.com/materialsproject/pymatgen/pull/3822/commits/4484d34a28d26874708b1f708a05a2227b7a55e5#r1623746333